@here/harp-olp-utils/README.md

```diff
@@ -0,0 +1,9 @@
+# @here/harp-olp-utils
+
+## Overview
+
+This module provides OLP utility classes.
```

Comment:
People outside HERE might not know OLP. Maybe add a link to platform.here.com

Reply:
Done.